app/views/layouts/application.html.erb

```diff
@@ -8,115 +8,86 @@
   <script src="//maps.google.com/maps/api/js?v=3.18&sensor=false&client=&key=&libraries=geometry&language=&hl=&region="></script>
   <script src="https://cdn.rawgit.com/googlemaps/v3-utility-library/master/markerclustererplus/src/markerclusterer_packed.js"></script>
   <script src="https://cdn.rawgit.com/googlemaps/v3-utility-library/master/infobox/src/infobox_packed.js"></script>
-  <meta name="viewport" content="width=device-width" />
 </head>
 <body>
-  <% if current_user && current_user.admin? %>
-    <nav class="navbar navbar-inverse navbar-fixed-top">
-      <div class="container-fluid">
-        <div class="navbar-header">
-          <a href="organizations#index" class="navbar-brand navbar-collapse" id="logo"> <%= image_tag("logo_white.png")%></a>
-          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-2">
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-        </div>
-        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-2">
-          <ul class="nav navbar-nav">
-            <li><%= link_to "Admin Panel: ", admin_organizations_path%> </li>
-            <li><%= link_to "Organizations", admin_organizations_path %></li>
-            <li><%= link_to "Events", admin_events_path %></li>
-            <li><%= link_to "News", admin_news_filters_path %></li>
-            <li><%= link_to "Users", admin_users_path %></li>
-            <li><%= link_to "Technologies", admin_technologies_path %></li>
-          </ul>
-        </div>
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container-fluid">
+      <a href="<%= root_path%>" class="navbar-brand" id="logo"> <%= image_tag("logo_white.png")%></a>
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-2">
```

Comment:
It looks like the `data-target` and connected `id` on line 24 were taken from example. I would rename them to something more descriptive of the element.